spec/features/practice_editor/publish_practice_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+describe 'Practice editor', type: :feature, js: true do
+  before do
+    @admin = User.create!(email: 'toshiro.hitsugaya@soulsociety.com', password: 'Password123', password_confirmation: 'Password123', skip_va_validation: true, confirmed_at: Time.now, accepted_terms: true)
+    @practice = Practice.create!(name: 'A public practice', slug: 'a-public-practice', approved: true, published: false)
+    @practice_partner = PracticePartner.create!(name: 'Diffusion of Excellence', short_name: '', description: 'The Diffusion of Excellence Initiative', icon: 'fas fa-heart', color: '#E4A002')
+    @admin.add_role(User::USER_ROLES[0].to_sym)
+  end
+
+  describe 'Publish practice flow' do
+    before do
+      login_as(@admin, :scope => :user, :run_callbacks => false)
+      visit practice_overview_path(@practice)
+      expect(page).to be_accessible.according_to :wcag2a, :section508
+      @save_button = find('#practice-editor-save-button')
+      @publish_button = find('#publish-practice-button')
+    end
+
+    it 'Should display an error modal that contains missing required fields if any exist' do
+      @adoptions = '13'
```

Comment:
probably don't need this to be an instance variable within the test
`@adoptions` -> `adoptions`